Network security: rebrand and new elastic cloud UX (#1785)
elastic/docs-content: deploy-manage/security/gcp-private-service-connect-traffic-filters.md

GCP Private Service Connect connects your Virtual Private Cloud (VPC) to the GCP-hosted services that you use, treating them as if they were in your VPC. You can create and use VPC endpoints to securely access GCP-hosted services.

You can also optionally filter traffic to your deployments by creating virtual private connection endpoint (VCPE) filters as part of your private connection policy in {{ecloud}}. This limits traffic to your deployment to the VCPE specified in the policy, as well as any other policies applied to the deployment.

Private Link is a connection between a Private Service Connect Endpoint and a Service Attachment. [Learn more about using Private Service Connect on Google Cloud](https://cloud.google.com/vpc/docs/private-service-connect#benefits-services).

To learn how private connection policies impact your deployment, refer to [](/deploy-manage/security/network-security-policies.md).

:::{tip}
{{ech}} also supports [IP filters](/deploy-manage/security/ip-filtering-cloud.md). You can apply both IP filters and private connections to a single {{ecloud}} resource.
:::

## Considerations

> * Private Service Connect filtering is supported only for Google Cloud regions.

Reviewer: ```suggestion
* Private connectivity with Private Service Connect is only supported in Google Cloud regions.
```
